lib/http/HttpClient_Apple.mm

```diff
@@ -50,9 +52,11 @@ void SendAsync(IHttpResponseCallback* callback)
         {
             m_callback = callback;
             NSString* url = [[NSString alloc] initWithUTF8String:m_url.c_str()];
-            NSURLSessionConfiguration* sessionConfig = [NSURLSessionConfiguration defaultSessionConfiguration];
             m_urlRequest = [[NSMutableURLRequest alloc] initWithURL:[NSURL URLWithString:url]];
-            m_session = [NSURLSession sessionWithConfiguration:sessionConfig];
+            if(m_session == nil) {
```

Comment:
This can lead to a raise condition

Reply:
This is a lazy initialisation. Moving the initialisation to the constructor of HttpRequestApple class.